build.sbt

```diff
@@ -63,7 +63,7 @@ libraryDependencies ++= Seq(
   "io.circe" %% "circe-optics" % "0.14.1",
   "io.circe" %% "circe-generic" % circeVersion,
   "io.circe" %% "circe-generic-extras" % "0.14.3",
-  "uk.gov.nationalarchives" %% "consignment-api-db" % "0.1.28",
+  "uk.gov.nationalarchives" %% "consignment-api-db" % "0.1.32",
```

Comment:
There is a new version `0.1.34` with the correct camel case naming of the new fields to use.